include/InstrumentMiscView.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * InstrumentMiscView.h - widget in instrument-track-window for setting up
+ *                        miscellaneous options not covered by other tabs
+ *
+ * Copyright (c) 2005-2014 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * Copyright (c) 2020 Martin Pavelek <he29.HS/at/gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef INSTRUMENT_MISC_VIEW_H
+#define INSTRUMENT_MISC_VIEW_H
+
+#include <QWidget>
+
+
+class ComboBox;
+class GroupBox;
+class InstrumentTrack;
+class LedCheckBox;
+
+
+class InstrumentMiscView : public QWidget
+{
+	Q_OBJECT
+public:
+	InstrumentMiscView(InstrumentTrack *it, QWidget *parent);
+	~InstrumentMiscView();
+
+	GroupBox *pitchGroupBox() {return m_pitchGroupBox;}
```

Comment:
Well, that is a never ending discussion. For both variables and functions, the only thing I'm sure of is that I definitely hate `type * thing`, but both `type* thing` and `type *thing` have good arguments for their use and they both make sense: one emphasizes that "thing" stores / returns a "pointer to type", the other that the data type stored / returned is a "type", but the "thing" is / returns just a pointer to it.

There seems to be no clear consensus on the interwebs, so I just stuck with what I first started using years ago and use it for both variables and functions, as mixing the styles would seem inconsistent to me.

But anyway: I'm guessing that when @JohannesLorenz pushes the button, clang-format will (should) pick one alignment and standardize the entire code-base (which currently uses all three alignments in places), and I'm not sure if it cares whether it is next to a variable or a function.

Reply:
Just checked, `clang-format` converts any `type* thing` into `type *thing`. We can still change that if we want. Anyways, it will change all to one defined style. So it does not matter what we do now.